src/plugins/workspace/public/components/workspace_creator/workspace_creator.tsx

```diff
@@ -177,7 +167,7 @@ export const WorkspaceCreator = (props: WorkspaceCreatorProps) => {
               savedObjects={savedObjects}
               onSubmit={handleWorkspaceFormSubmit}
               operationType={WorkspaceOperationType.Create}
-              permissionEnabled={isPermissionEnabled}
+              permissionEnabled={false}
```

Comment:
Can we remove this props instead of giving false.

Reply:
You're right, let's remove the permissionEnabled property. I met a unit tests failed without this property before. But seems all the unit tests can be passed now. Let me remove this property.